README.md

```diff
@@ -0,0 +1,24 @@
+This is a boilerplate extension for experiments. It *is not* designed to work out of the box. It *is* designed to be:
+* something you can clone from this repository
+* alter some key values
+* create a simple experiment that you can use elsewhere
+
+To use (on Linux or OS X):
+
+```
+git clone https://github.com/web-ext-experiments/boilerplate-experiment
+cd boilerplate-experiment
+rm -r .git
+```
+
+This will clone the repository and remove the .git configuration so that you can use your own source control.
+
+Next steps:
+* `api.js` is where the code for your API is added.
```

Comment:
I would probably do it in the opposite, but everyone does it differently, i'll put something like "in no specific order..."